tests/unit/reconcile/connectors/test_snowflake.py

```diff
@@ -0,0 +1,219 @@
+import re
+from unittest.mock import MagicMock, create_autospec
+
+import pytest
+from databricks.sdk import WorkspaceClient
+from pyspark.errors import PySparkException
+
+from databricks.labs.remorph.reconcile.connectors.snowflake import SnowflakeDataSource
+from databricks.labs.remorph.reconcile.constants import SourceDriver
+from databricks.labs.remorph.reconcile.recon_config import JdbcReaderOptions, Table
+
+
+@pytest.fixture
+def snowflake_data_source():
+    # Create a mock SparkSession
+    pyspark_sql_session = MagicMock()
+    spark = pyspark_sql_session.SparkSession.builder.getOrCreate()
+
+    # Define the source, workspace, and scope
+    source = "snowflake"
+    ws = create_autospec(WorkspaceClient)
+    scope = "scope"
+
+    SnowflakeDataSource_ = SnowflakeDataSource(source, spark, ws, scope)
```

Comment:
```suggestion
    ds = SnowflakeDataSource(source, spark, ws, scope)
```
Use proper snake_case for variable names

Reply:
done